src/include/gpopt/translate/CTranslatorDXLToPlStmt.h

```diff
@@ -137,6 +137,12 @@ class CTranslatorDXLToPlStmt
 	// command type
 	CmdType m_cmd_type;
 
+	// is dml operation requiring returning
```

Comment:
The comment sounds a bit bulky, I suggest to re-write it, smth like:
```suggestion
	// does dml operation require returning
```

Reply:
fixed it